install/conda.rst

```diff
@@ -25,6 +25,10 @@ If not, you can quickly get started by following Continuum's official installati
 - `Install Anaconda <https://www.continuum.io/downloads>`__ if you want a complete, science-ready Python installation with minimal setup.
 - Otherwise, `install the leaner Miniconda <http://conda.pydata.org/miniconda.html>`__ version and install packages as you need them.
 
+.. warning:: 
+
+   You need the Python 2 (eg. Miniconda2) distribution. If you want to use the experimental Python 3 support, you should try our source installation. 
```

Comment:
Well we sure ain't making them for this release, because it's gotta get shipped already. @jmatt there is no need to be doing work outside work. If we need 3.0 conda binaries we'll schedule it.